### ISSUE TITLE
Not on same device error in lightly 1.5.11 

### ISSUE THREAD

**Issue by sophiamaedler** [lightly-ai/lightly#1630]
I am using the DINOLoss  function implemented in lightly `from lightly.loss import DINOLoss.` 

On version 1.5.11 I get the following error:
```
  File "../lib/python3.10/site-packages/lightly/loss/dino_loss.py", line 134, in forward
    t_out = F.softmax((teacher_out - self._center.value) / teacher_temp, dim=-1)
RuntimeError: Expected all tensors to be on the same device, but found at least two devices, cuda:0 and cpu!
```
running the exact same code but with version 1.5.6 (the rest of the env is constant) everything runs fine. 

operating system: 
```
NAME="SLES"
VERSION="15-SP4"
VERSION_ID="15.4"
PRETTY_NAME="SUSE Linux Enterprise Server 15 SP4"
ID="sles"
ID_LIKE="suse"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:suse:sles:15:sp4"
DOCUMENTATION_URL="https://documentation.suse.com/"
```

Package versions:
Python: 3.10.14
PyTorch: 2.4.0
PyTorch Lightning: 2.4.0

I unfortunately don't have time to create a minimal working example at the moment, but I still wanted to report since maybe its a quick fix  and you already know what the problem might be.  If not I can  follow up with a minimal example in a couple of weeks. 

Cheers
Sophia

**Comment by guarin** [comment]
Hi, thanks for the issue report! This bug was introduced in 1.5.11, there is an open PR to fix it here: #1629. I recommend to revert to 1.5.10 for now.

**Comment by guarin** [comment]
Fix is on master branch and will be released with 1.5.12
